src/ol/control/ScaleLine.js

```diff
@@ -51,6 +51,8 @@ const DEFAULT_DPI = 25.4 / 0.28;
  * @property {string} [className='ol-scale-line'] CSS Class name.
  * @property {number} [minWidth=64] Minimum width in pixels at the OGC default dpi. The width will be
  * adjusted to match the dpi used.
+ * @property {number|undefined} [maxWidth=undefined] Maximum width in pixels at the OGC default dpi. The width will be
```

Comment:
```suggestion
 * @property {number} [maxWidth] Maximum width in pixels at the OGC default dpi. The width will be
```
This is a shorter equivalent for an optional property, which implies `undefined`.